src/tranqu/transpiler/transpiler_manager.py

```diff
@@ -24,6 +28,40 @@ class TranspilerManager:
 
     def __init__(self) -> None:
         self._transpilers: dict[str, Any] = {}
+        self._default_transpiler_lib: str | None = None
+
+    def register_default_transpiler_lib(
```

Comment:
Should `register_default_transpiler_lib` throw an exception when a default transpiler is already registered?
There might be use cases where we want to call `register_default_transpiler_lib` multiple times - for example, 
when we want to transpile 10 circuits / times, using Qiskit for the first 5 and Tket for the remaining 5.